dotnet/src/Functions/Functions.OpenAPI/Model/RestApiOperationParameter.cs

```diff
@@ -52,12 +52,18 @@ public sealed class RestApiOperationParameter
     /// </summary>
     public string? DefaultValue { get; }
 
+    /// <summary>
+    /// Specifies whether arrays and objects should generate separate parameters for each array item or object property.
+    /// </summary>
+    public bool Explode { get; }
```

Comment:
nit: 
```suggestion
    public bool Expand { get; }
```

Explode to me reads as related to throwing exceptions.

Reply:
Yep, np, I just used the name OpenAPI model uses. Fixed.